Guide.docc/CommonProblems.md

```diff
@@ -393,3 +393,312 @@ struct LivingIsland: Feedable {
 Here, a new type has been created that can satisfy the needed inheritance.
 Incorporating will be easiest if the conformance is only used internally by
 `Island`.
+
+## Crossing Isolation Boundaries
+
+Any value that needs to move from one isolation domain to another
+must be `Sendable`.
+Using non-`Sendable` values in contexts that require `Sendable` types is a very
+common problem.
+And because libraries and frameworks may be updated to use Swift's
+concurrency features, these issues can come up even when your code hasn't
+changed.
+
+### Implicitly-Sendable Types
+
+Many value types consist entirely of `Sendable` properties.
+The compiler will treat types like this as implicitly `Sendable`, but _only_
+when they are non-public.
+
+```swift
+public struct ColorComponents {
+    public let red: Float
+    public let green: Float
+    public let blue: Float
+}
+
+@MainActor
+func applyBackground(_ color: ColorComponents) {
+}
+
+func updateStyle(backgroundColor: ColorComponents) async {
+    await applyBackground(backgroundColor)
+}
+```
+
+Despite `ColorComponents` being eligible for an implicit `Sendable`
+conformance, this code will result in the following error:
```

Comment:
I think we should say _why_ a `Sendable` conformance isn't inferred: `Sendable` is part of the public API contract. Public types must state an explicit `Sendable` conformance so that the compiler doesn't make any implicit API promises that you didn't intend to make.

Reply:
yeah for sure